### PR TITLE
pull request

### PR DESCRIPTION
**Pull request by akpathan2799**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] You're not using Vercel or Netlify.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by akpathan2799** [comment]
can you please check it

**Comment by akpathan2799** [comment]
can you please check it

File: domains/adeebkhan.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/akpathan2799/akpathan2799.github.io",
+    "owner": {
+        "username": "akpathan2799",
+        "email": "akpathan2799@gmail.com",
```
Comment (CuteDog5695) [inline domains/adeebkhan.json:6]: remove comma at line 6

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by akpathan2799** [comment]
check it now

**Comment by akpathan2799** [comment]
check it now

**Pull request closed without merging**